docs/pages/admin-guides/management/admin/logging.mdx

```diff
@@ -0,0 +1,71 @@
+---
+title: Logger configuration
```

Comment:
```suggestion
title: Logger Configuration
```
We use title case in titles (here's the style guide: https://github.com/gravitational/docs/blob/main/docs-contributors/style-guide.md)